scripts/orchestrators/setup-benchpress.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
```

Comment:
Instead of constantly jumping to `pwsh` shells, wouldn't it be better to have the whole benchpress setup logic in a powershell file and just invoke one `pwsh` command?

Reply:
`setup-xxx` scripts are all in bash format, I don't want to make an exception for benchpress and have it in powershell format.

in the `setup-benchpress.sh`, we do checks to find,

- if benchpress itself is installed or not
- if all dependencies are installed or not

if they're not installed, they get installed by the script, if they already installed, we just log it and continue

if we merge all powershell scripts into one, we can't check individual dependencies are installed or not

Reply:
Yeah, I get that. What I was thinking was that `setup-benchpress.sh` could be just a bootstrapper for a `setup-benchpress.ps1` and you can run everything from that single powershell script, instead of keep calling `pwsh`. Because right now you're invoking `pwsh` to check stuff, but the logic itself is in `bash`. Don't love the mix, and that's why I thought of having a single script that does the whole benchpress setup would be a better idea.

Reply:
make sense, done!